conf/knnaas_config.yaml

```diff
@@ -0,0 +1,16 @@
+# Config file for KNN as a Service
+
+
+# Port number on which to run the service. (Required)
+port: 9090
+
```

Comment:
We don't need to be concerned about the port number. User is going to change the config as they need.